algosdk/mnemonic.py

```diff
@@ -92,6 +94,21 @@ def to_public_key(mnemonic):
     return encoding.encode_address(key.verify_key.encode())
 
 
+def to_address(mnemonic):
+    """
+    Return the address for the mnemonic of the public key.
+
+    Args:
+        mnemonic (str): mnemonic of the public key
```

Comment:
@jasonpaulos I followed the description of `to_public_key` when writing this. I realized that this might be a little confusing. The mnemonic is actually the concatenation of the private key and the public key, as seen in `generate_account` [here](https://github.com/algorand/py-algorand-sdk/blob/develop/algosdk/account.py#L16). For most of the functions in `mnemonic.py`, I see snippets of "mnemonic of the public key" or "mnemonic of private key". I'm wondering if it should just be renamed and redocumented in general as simply "mnemonic" to reduce confusion.